xls/examples/hack.x

```diff
@@ -0,0 +1,258 @@
+// Copyright 2021 The XLS Authors
```

Comment:
Naming nit: How about calling this hack_cpu.x? As is, the name is mystifying.

Reply:
Done